css/place-content/demos/interactive-parameters/index.html

```diff
@@ -0,0 +1,200 @@
+      height: 150px;
+      gap: 20px;
+    }
+
+    .container div {
+      width: 25px;
+      height: 25px;
+    }
+
+    .flex-layout {
+      display: flex;
+      flex-wrap: wrap;
+    }
+
+    .flex-layout div {
+      background-color: #2E9AFF;
+    }
+
+    .grid-layout {
+      display: grid;
+    }
+
+    .grid-layout div {
+      background-color: #F498AD;
+    }
+
+    @media (max-width: 768px) {
+      body {
+        padding: 30px;
+      }
+    }
+  </style>
+</head>
+<body>
+  <div class="select-wrapper">
+    <div class="align-parameters">
+      <label for="align-select">align-content</label>
+      <select id="align-select">
+        <option value="start" selected>start</option>
+        <option value="end">end</option>
+        <option value="center">center</option>
+        <option value="flex-start">flex-start</option>
+        <option value="flex-end">flex-end</option>
+        <option value="space-between">space-between</option>
+        <option value="space-around">space-around</option>
+        <option value="space-evenly">space-evenly</option>
+        <option value="baseline">baseline</option>
+      </select>
+    </div>
+    <div class="justify-parameters">
+      <label for="justify-select">justify-content</label>
+      <select id="justify-select">
+        <option value="start" selected>start</option>
+        <option value="end">end</option>
+        <option value="left">left</option>
+        <option value="right">right</option>
+        <option value="center">center</option>
+        <option value="flex-start">flex-start</option>
+        <option value="flex-end">flex-end</option>
+        <option value="space-between">space-between</option>
+        <option value="space-around">space-around</option>
+        <option value="space-evenly">space-evenly</option>
+      </select>
+    </div>
+    <div class="code-example">
+      <code>place-content: start start;</code>
+    </div>
+  </div>
+  <div class="container flex-layout">
+    <div></div>
+    <div></div>
+    <div></div>
+  </div>
+  <div class="container grid-layout">
+    <div></div>
+    <div></div>
+    <div></div>
+  </div>
+  <script>
+    const alignSelect = document.querySelector('#align-select')
+    const justifySelect = document.querySelector('#justify-select')
+    const codeExample = document.querySelector('code')
+    const container = document.querySelectorAll('.container')
+    let firstParam = alignSelect.value
+    let secondParam = justifySelect.value
+
+    alignSelect.addEventListener('change', styleChanger)
+    justifySelect.addEventListener('change', styleChanger)
+
+    function styleChanger(event) {
+      if (event.target.id == 'align-select') {
+        firstParam = this.value
+      } else {
+        secondParam = this.value
+      }
+
+      for (let item of container) {
+        item.style.placeContent = `${firstParam} ${secondParam}`
+      }
+
+      codeExample.textContent = `place-content: ${firstParam} ${secondParam};`
```

Comment:
```suggestion
      if (firstParam === secondParam) {
        codeExample.textContent = `place-content: ${firstParam};`
      } else {
        codeExample.textContent = `place-content: ${firstParam} ${secondParam};`
      }
```

А что если нам добавить проверку на одинаковость параметров и, если они одинаковые, то выводить только одно значение? Как принято писать в жизни.

Reply:
Будет супер 😊